### PR TITLE
chore: restructure and add updated notebooks

### PR DESCRIPTION
**Pull request by amangoel185**
Modify #5 according to the new naming scheme:

- Rename the folders to reflect episode names
- Delete and add the notebooks with updated links and names

### CODE REVIEW

File: ep04-mplhep.ipynb [hunk 1]
```diff
@@ -7,7 +7,7 @@
         "colab_type": "text"
       },
       "source": [
-        "<a href=\"https://colab.research.google.com/github/plttraining/hsf_matplotlib_notebooks/blob/main/Higgs_4leptons.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
```
Comment (amangoel185) [inline ep04-mplhep.ipynb:10]: We can change the links to point at `plttraining` after merging. If that doesn't work, someone with access to `plttraining` can manually add the notebooks via Google Colab.

Comment (GuillermoFidalgo) [reply ep04-mplhep.ipynb:10]: Ok, as long as we remember to point to the correct url for each notebook, we should be good.
I can give you access to the repo so you can do it and not wait for my approval.

Comment (amangoel185) [reply ep04-mplhep.ipynb:10]: Alright, I can do that

File: ep05-dimuonspectrum-awkward.ipynb [hunk 2]
```diff
@@ -7,7 +7,7 @@
         "colab_type": "text"
       },
       "source": [
-        "<a href=\"https://colab.research.google.com/github/plttraining/hsf_matplotlib_notebooks/blob/main/CMS-Dimuons.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
+        "<a href=\"https://colab.research.google.com/github/amangoel185/hsf_matplotlib_notebooks/blob/amangoe185%2Fchore%2Frestructure/ep05-dimuonspectrum-awkward.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
```
Comment (plttraining) [inline ep05-dimuonspectrum-awkward.ipynb:10]: Note: 
Also has to point to correct url after merging.

File: ep05-dimuonspectrum-pandas.ipynb [hunk 3]
```diff
@@ -7,7 +7,7 @@
         "colab_type": "text"
       },
       "source": [
-        "<a href=\"https://colab.research.google.com/github/plttraining/hsf_matplotlib_notebooks/blob/main/ep05_CMS_Dimuons_pandas.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
+        "<a href=\"https://colab.research.google.com/github/amangoel185/hsf_matplotlib_notebooks/blob/amangoe185%2Fchore%2Frestructure/ep05-dimuonspectrum-pandas.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
```
Comment (plttraining) [inline ep05-dimuonspectrum-pandas.ipynb:10]: Same here

**Review by plttraining — APPROVED** [review]

**Pull request merged**